modules/ROOT/pages/partner-manager-identifiers.adoc

```diff
@@ -80,34 +80,34 @@ Use the following values to configure the X12 GS identifier type:
 |Setting |Description | Required
 
 |Identifier value
-|EDI X12-GS ID for your trading partner, which can be up to 15 characters. Your trading partner supplies you with the proper EDI X12 Group Sender and Receiver IDs during onboarding.
+|EDI X12-GS ID for your partner, which can be up to 15 characters. Your partner supplies you with the proper EDI X12 Group Sender and Receiver IDs during onboarding.
 |Yes
 |===
 
 [[reference-id]]
 == Outbound Message Flow Partner Reference Identifiers
 
-Outbound message flows that transform your backend application messages to EDI format use the partner Reference ID when sending data to your trading partners.
+Outbound message flows that transform your backend application messages to EDI format use the partner Reference ID when sending data to your partners.
 
 Unlike EDI transactions that have well-defined standards for how sender, receiver, and document type information is presented in the data, the source messages for outbound B2B data exchange originating from your enterprise backend systems typically follow schemas. These schemas are usually defined by your backend application teams according to your organization’s data models.
 
-The receiving trading partner information about the application message from the backend application might not necessarily be the same as the name of the partner in Partner Manager. This scenario is similar to how ISA and GS identifiers on an inbound EDI message might not be the same as the profilename of the partner.
+The receiving partner information about the application message from the backend application might not necessarily be the same as the name of the partner in Partner Manager. This scenario is similar to how ISA and GS identifiers on an inbound EDI message might not be the same as the profilename of the partner.
 
 Partner Manager’s outbound message processing framework makes a dynamic lookup to determine the receiving partner based on the content in the application message received from the backend systems.
 
 === Example
 
-In this example, an outbound purchase order XML message from your ERP system to your trading partner Mythical Supplier has the following content in the payload:
+In this example, an outbound purchase order XML message from your ERP system to your partner Mythical Supplier has the following content in the payload:
```

Comment:
In this example, an outbound purchase order XML message from your ERP system to your partner, Mythical Supplier, has the following content in the payload: